src/shibuya.js

```diff
@@ -28,24 +28,24 @@ const main = async () => {
     const proxyOnShibuya = shibuyaHelper.getProxyAccount(TURING_PARA_ID, shibuyaAddress);
 
     console.log(`\na) Add a proxy of Turing (paraId:${TURING_PARA_ID}) for Alice on Shibuya ...\n Proxy address: ${proxyOnShibuya}\n`);
-    await sendExtrinsic(shibuyaHelper.api, shibuyaHelper.api.tx.proxy.addProxy(proxyOnShibuya, 'Any', 0), keyPair);
+    await sendExtrinsic(shibuyaHelper.api, shibuyaHelper.api.tx.proxy.addProxy(proxyOnShibuya, 'Any', 0), keyPair); 
 
     console.log('\nb) Topping up the proxy account on Shibuya with SBY ...\n');
-    const transferSBY = shibuyaHelper.api.tx.balances.transfer(proxyOnShibuya, '1000000000000000000000');
+    const transferSBY = shibuyaHelper.api.tx.balances.transfer(proxyOnShibuya, '1000000000000000000000'); 
```

Comment:
Nobody can tell how many tokens is this. Always use BN number to calculate the amount.

For example, if you want to send 100 SBY here.
```
const amount = new BN(100, 10);
const sbyAmountBN = amount.mul(getDecimalBN(SHIBUYA_DECIMAL));

// Then, use sbyAmountBN.toString()
```